example/android/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,2 @@
+connection.project.dir=
+eclipse.preferences.version=1

```

Comment:
Please remove this file (as well as `example/android/.project`) - They should not be in source control.

Reply:
oh, my bad. ok. sorry, I don't know why they appear every-time

Reply:
The reason they (re-)appear is a buggy Java plugin for VS Code. Unless you heavily rely on it, the recommendation is to uninstall it.
Also see: https://github.com/redhat-developer/vscode-java/issues/618